src/examples/PDFExamplePage.tsx

```diff
@@ -0,0 +1,55 @@
+'use strict';
+import {View} from 'react-native';
+import React from 'react';
+import {Example} from '../components/Example';
+import {Page} from '../components/Page';
+import Pdf from 'react-native-pdf';
+
+export const PDFExamplePage: React.FunctionComponent<{}> = () => {
+  const example = `import Pdf from 'react-native-pdf';
+function Example() {
+  const source = { uri:"ms-appx:///dummy.pdf" }
+  const state = { width: 800, height: 600 }
+  return (
+    <View style={{flex:1, width: state.width, height: state.height}}>
+      <Pdf
+        source={source} 
+        scale={state.scale}
+        horizontal={state.horizontal}
+        style={{flex:1}}/>
+    </View>
+  );
+}`;
```

Comment:
We just made a decision that we want the code excerpts to just be the JSX snippet. Could you remove the import line and in the Pdf component instance use the raw input values rather than the variables? Should look something like this:

'''
<View style={{flex:1, width: 800, height: 800}}>
      <Pdf
        source={{uri:"ms-appx:///dummy.pdf"}} 
        scale={state.scale} // replace with actual data
        horizontal={state.horizontal} // replace with actual data
        style={{flex:1}}/>
    </View>
'''